internal/hcp/registry/hcl.go

```diff
@@ -127,9 +141,12 @@ func NewHCLMetadataRegistry(config *hcl2template.PackerConfig) (*HCLMetadataRegi
 		bucket.RegisterBuildForComponent(source.String())
 	}
 
+	ui.Say(fmt.Sprintf("Tracking build on HCP Packer with fingerprint %q", bucket.Iteration.Fingerprint))
```

Comment:
Is there another way to reuse the content of the IterationStatusSummary without having to copy it into this call. The messaging below is helpful.  
```
You may resume work on this iteration in further Packer builds by defining the following variable in your environment:
HCP_PACKER_BUILD_FINGERPRINT="generated-fingerprint"
```

Reply:
The messaging you reference is useful when the iteration fails, but I'm not sure we should reuse the same phrasing for notifying users of which iteration we are tracking their build on at the beginning of the build.
I'm not sure it'd be worth it to split the `IterationStatusSummary` function to factorise such a small amount of code tbh, I'd be inclined to leave it as-is, unless you have a better idea?